src/mercury_engine_data_structures/formats/bmdefs.py

```diff
@@ -0,0 +1,100 @@
+import construct
+from construct import (
+    Const,
+    Construct,
+    Enum,
+    Float32l,
+    Int8ul,
+    Int16ul,
+    Int32ul,
+    Struct,
+)
+
+from mercury_engine_data_structures.common_types import StrId, make_vector
+from mercury_engine_data_structures.formats import BaseResource
+from mercury_engine_data_structures.game_check import Game
+
+Status = Enum(
+    Int8ul,
+    DISABLED=0,
+    ENABLED=1,
+)
+
+EnemiesStruct = Struct(
```

Comment:
Should be renamed to singular `EnemyStruct` because "one thing" of this is the struct for one enemy.